test/hal_to_clouddriver_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The Spinnaker Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package test
+
+import (
+	"bytes"
+	"github.com/ghodss/yaml"
+	"github.com/spinnaker/kleat/api/client"
+	"github.com/spinnaker/kleat/internal/parse"
+	"io/ioutil"
+	"testing"
+)
+
+func TestHalToClouddriver(t *testing.T) {
+	h, err := parse.ParseHalConfig("./data/halconfig.yml")
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	gotC, err := parse.HalToClouddriver(*h)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	wantC, err := parseClouddriverConfig("./data/clouddriver.yml")
```

Comment:
Based on my research, "got" and "want" are words you need to use in order to make cloud-native assertions.